projects/packages/jetpack-mu-wpcom/src/features/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-launchpad.php

```diff
@@ -146,6 +146,7 @@ public function get_data( $request ) {
 			'checklist'          => wpcom_get_launchpad_checklist_by_checklist_slug( $checklist_slug ),
 			'is_enabled'         => wpcom_get_launchpad_task_list_is_enabled( $checklist_slug ),
 			'is_dismissed'       => wpcom_launchpad_is_task_list_dismissed( $checklist_slug ),
+			'title'              => wpcom_get_launchpad_checklist_title_by_checklist_slug( $checklist_slug ),
```

Comment:
As a general thought, it may be worth building one instance of the task list and then calling functions -- while this pattern is clean, it does feel like we may end up doing duplicate work.
(Definitely not blocking - even if we tackle this, I think it should be in a follow-up.)